XCUITests/SaveLoginsTests.swift

```diff
@@ -5,36 +5,95 @@
 import XCTest
 
 let domain = "http://localhost:\(serverPort)"
+let domainLogin = "test@example.com"
+let domainSecondLogin = "test2@example.com"
 let testLoginPage = path(forTestPage: "test-password.html")
+let testSecondLoginPage = path(forTestPage: "test-password-2.html")
 let savedLoginEntry = "test@example.com, http://localhost:\(serverPort)"
 let urlLogin = "linkedin.com"
 let mailLogin = "iosmztest@mailinator.com"
+//The following seem to be labels that change a lot and make the tests break; aka volatile. Let's keep them in one place.
+let loginsListURLLabel = "Website, \(domain)"
+let loginsListUsernameLabel = "Username, test@example.com"
+let loginsListPasswordLabel = "Password"
+let defaultNumRowsLoginsList = 2
+let defaultNumRowsEmptyLoginsList = 0
 
 class SaveLoginTest: BaseTestCase {
 
-    private func saveLogin() {
-        navigator.openURL(testLoginPage)
+    private func saveLogin(givenUrl: String) {
+        navigator.openURL(givenUrl)
         waitUntilPageLoad()
         waitForExistence(app.buttons["submit"], timeout: 3)
         app.buttons["submit"].tap()
         app.buttons["SaveLoginPrompt.saveLoginButton"].tap()
     }
 
     private func openLoginsSettings() {
+        navigator.goto(SettingsScreen)
+        navigator.goto(LoginsSettings)
+        waitForExistence(app.tables["Login List"])
+    }
+    
+    func testLoginsListFromBrowserTabMenu() {
+        waitForTabsButton()
+        //Make sure you can access empty Login List from Browser Tab Menu
+        navigator.goto(LoginsSettings)
+        waitForExistence(app.tables["Login List"])
+        XCTAssertTrue(app.searchFields["Filter"].exists)
+        XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsEmptyLoginsList)
+        saveLogin(givenUrl: testLoginPage)
+        //Make sure you can access populated Login List from Browser Tab Menu
         navigator.goto(LoginsSettings)
         waitForExistence(app.tables["Login List"])
+        print(app.debugDescription)
+        XCTAssertTrue(app.searchFields["Filter"].exists)
+        XCTAssertTrue(app.staticTexts[domain].exists)
+        XCTAssertTrue(app.staticTexts[domainLogin].exists)
+        XCTAssertEqual(app.tables["Login List"].cells.count, 3)
+    }
+    
+    func testPasscodeLoginsListFromBrowserTabMenu() {
+        navigator.performAction(Action.SetPasscode)
+        userState.newPasscode = "222222"
```

Comment:
This is not used, the passcode taken is 11111 the default one in FxScreenGraph, with that is enough, no need to add a new one I guess